css/about.css

```diff
@@ -32,8 +32,12 @@ Main Content
 ===================================
 */
 /* wrapper spacing similar to section */
+
+main {
```

Comment:
key issue with global wrapper adjustments - vertical padding was initially applied onto wrappers, overwriting global wrapper adjustments from `events-page` pr. the newly added wrapper styles changed the max width to 100%, which caused wrapper to not apply